client/source/components/RR/ReviewList/AddReview/RadioInput.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+
+  <RadioInput
+    labelText="Do you recommend this product? *"
```

Comment:
Future styling option could be to make * requiring input a different color like red - though this is obviously low priority, it looks great!

Reply:
Yes, maybe some error handling logic. Right now I'm deferring to validation via the attribute required.